packages/admin-panel/src/table/columnTypes/QrCodeButton.js

```diff
@@ -0,0 +1,49 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { openQrCodeModal } from '../../qrCode';
+import { IconButton } from '../../widgets';
+
+const LittleQrCodeIcon = () => (
```

Comment:
MUI has a few options for QR Code Icons if liked any of them? https://mui.com/material-ui/material-icons/?query=qr+code

Reply:
Good spotting, unfortunately not in the version of mui we're on. Happy for it to stay custom for now?

Reply:
Sure ting 👍